logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -27,47 +27,63 @@
 # /appengine-vmruntime/vmruntime/cloud_logging.py
 
 import logging.handlers
-import os
 
-from google.cloud.logging.handlers._helpers import format_stackdriver_json
+from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers.transports import BackgroundThreadTransport
+from google.cloud.logging.resource import Resource
 
-_LOG_PATH_TEMPLATE = '/var/log/app_engine/app.{pid}.json'
-_MAX_LOG_BYTES = 128 * 1024 * 1024
-_LOG_FILE_COUNT = 3
+DEFAULT_LOGGER_NAME = 'python'
```

Comment:
I *think* for the gae logs the name is supposed to be ''projects/{}/logs/appengine.googleapis.com%2Fapp.shutdown'.format(PROJECT_ID) . May or may not be necessary to get logs show up in right place.